app/assets/stylesheets/db/controllers/edit-requests.scss

```diff
@@ -0,0 +1,44 @@
+body.db-edit_requests-index {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector `db-edit_requests-index` should be written in lowercase with hyphens